linkerd/grpc-route/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
```

Comment:
am i losing my mind or does this crate not have a `lib.rs`? i can't seem to find it in the diff...

Reply:
crate should have been removed